docs/querying/api.md

```diff
@@ -750,4 +752,4 @@ This takes no parameters or body.
 $ curl -XPOST http://localhost:9090/api/v1/admin/tsdb/clean_tombstones
 ```
 
-*New in v2.1*
+*New in v2.1*
```

Comment:
The notes are generally put here (presuming we're following Python), so it'd be PUT is new in v2.9 or similar.

Reply:
Fixed